README.md

```diff
@@ -242,10 +242,6 @@ Yes it is! This plugin comes with .po and .mo files. It is already translated in
 * Fixed: Improved performance of the field replacement functionality
 * Fixed: Reduced the frontend data render calls to help with performance
 
-### 1.2.4 ###
```

Comment:
Why have we deleted this?

Reply:
In the master branch, there was two same changelog - https://d.pr/i/7GiH3W